src/Microsoft.AspNetCore.Http.Connections/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -182,6 +182,14 @@ public WebSocketsTransport(WebSocketOptions options, IDuplexPipe application, Ht
                     }
                 }
             }
+            catch (WebSocketException ex) when (
+                ex.ErrorCode == 997 ||
+                // Sometimes this error is raised without the ErrorCode
+                ex.Message == "The remote party closed the WebSocket connection without completing the close handshake.")
```

Comment:
There's a specific websocket error code to look for.

Reply:
I know, 997, the line above. I found that sometimes it isn't set

Reply:
There's another property called WebsocketErrorCode

Reply:
That is also not set

Reply:
People getting this issue: https://github.com/aspnet/AspNetCoreModule/issues/77

Reply:
I don’t want this text comparison in our code. We control the websocket implementation. If it doesn’t set the right flag, we should fix it. It looks like it does so let’s use that.

Reply:
Should the error be ignored at all? Are WebSocketsTest and WebSocketsReceivesAndSendsPartialFramesTest written correctly? Or is it a bug in the framework? https://github.com/aspnet/AspNetCoreModule/issues/77

Reply:
Yes it should be ignored. It happens for example if an desktop application closes. We basically want to ignore errors that can happen if a client disconnects